src/main/java/org/jenkinsci/test/acceptance/plugins/artifactory/ArtifactoryPublisher.java

```diff
@@ -31,5 +31,7 @@ public ArtifactoryPublisher(Job job, String path) {
     public void refresh() {
         control("details/validate-button").click();
         waitFor(driver, hasContent("Items refreshed successfully"), 10);
+        // Wait for the refresh to populate the options
+        elasticSleep(1000);
```

Comment:
I was not able to figure it out since it seems to use a control I am familiar with...